lib/util/storage/database/wallet_repository.dart

```diff
@@ -51,7 +51,7 @@ class WalletRepository extends _WalletRepository {
     DatabaseClient databaseClient,
   ) async {
     try {
-      await _store.record(wallet.id).add(databaseClient, _mapWallet(wallet));
+      await _store.record(wallet.id).put(databaseClient, _mapWallet(wallet));
```

Comment:
Why adding a default name in the form affects how we store the wallet in the db?

Reply:
this is for updating the name when we import an already existing wallet. 
add fails if it already exists, put forces it into the database